march_hardware/src/EtherCAT/EthercatSDO.cpp

```diff
@@ -26,10 +26,13 @@ int sdo_bit8_write(uint16_t slave, uint16_t index, uint8_t sub, uint8_t value)
   return received_working_counter;
 }
 
-int sdo_bit8_read(uint16_t slave, uint16_t index, uint8_t sub, int* val_size, uint8_t* value)
+/**
+ * This function returns a working counter. The actual 8 bit read value can be found in the parsed argument; value
```

Comment:
```suggestion
 * This method returns a working counter. The actual 8 bit read value can be found in the parsed argument; value
```

Reply:
I always thought it was a conventional difference between Python and c++

Reply:
My mistake, not necessarily a difference between languages, but rather a difference between OOP and regular programming. Because this piece of code is not specific to an object, it is not a method.

Reply:
O wow I learned something new today